mediapipe/__init__.py

```diff
@@ -1,13 +1,13 @@
-# Copyright 2019 - 2022 The MediaPipe Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+
+from mediapipe.python import *
+import mediapipe.python.solutions as solutions 
+import mediapipe.tasks.python as tasks
+
+
+del framework
```

Comment:
why we need this?

Reply:
This is auto generated with pip install . . I have added this here and removed the line from setup.py so that it is not generated every time we run pip install . . Otherwise the file needs to be cleaned up every time.